src/Utils/browsercheck.ts

```diff
@@ -34,6 +44,20 @@ function samsung(range: string, userAgent: string) {
 }
 
 export function browserNotCompatible() {
-  const userAgent = (window.navigator.userAgent || '').toLowerCase();
-  return safari('<13', userAgent) || samsung('<9', userAgent) || !Object.values || !window.Intl || !window.crypto;
+  const userAgent = (window.navigator.userAgent || "").toLowerCase();
+  return (
+    safari("<13", userAgent) ||
+    samsung("<9", userAgent) ||
+    !Object.values ||
+    !window.Intl ||
+    !window.crypto
+  );
+}
+
+export function browserPayNotCompatible(): boolean {
```

Comment:
This new function is not used, is it? So I do not understand why this MR is changing this file.

Reply:
This seems to have been left behind after a revert commit. Have removed it.